packages/shared/src/types/experiments.types.ts

```
@@ -0,0 +1,114 @@
+import type {AccountId} from '@shared/types/accounts.types';
+import type {Environment} from '@shared/types/environment.types';
+import {BaseStoreItem} from '@shared/types/utils.types';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)